ush/parsing_namelists_MOM6.sh

```diff
@@ -51,6 +51,15 @@ elif [ $OCNRES = '100' ]; then
   CHLCLIM="seawifs_1998-2006_smoothed_2X.nc"
   MOM6_RESTART_SETTING='n'
   MOM6_RIVER_RUNOFF='False'
+elif [ $OCNRES = '400' ]; then
+  NX_GLB=90
```

Comment:
@jiandewang can you confirm these settings for @WalterKolczynski-NOAA ?

Reply:
we can consider double the time steps or at least double the DT_DYNAM, note this kind of super coarse resolution ocean model will be unstable